README.md

```diff
@@ -13,3 +14,38 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+## My Response
+
+1.
+The first is that when we are analyzing an algorithm we ignore any constants that might arise, this can change things significantly.  On a lower level, $x^2$ and $4x^2$ are not the same function even though the would have the same complexity, $4x^2$ increases faster than its parent function.
+
+The second is that the complexity of an algorithm doesn't take into account the hardware of the machine running the program.  It could run faster or slower based on the resources available.
+
+Cade brought up this one while I was talking to him in lab.  The bounds of our complexity can come into question for instance a program might have upper and lower bound that are $2^n$ and $n$ respectively, which means we run into very inconsistent run times, you might run the program a couple times and it is very quick more in line with the lower bound but then when you run it a third time it could be there for an hour reflecting the upper bound.
```

Comment:
Don't understand this. The asymptotic complexity is a single expression; where do those bounds come from?

Reply:
oh ok, I believe understand now, I had to review it online I need to take time over break to fully go over some of this because it feels like some of it should have been in my discrete structures class when it wasn't.  The upper $O$ bound is loose, so for something like a binary search, it's has an upper bound $O(n)$ but also has imprecise bounds such as $n^2$ and $n^3$, even though none of those would be appropriate to the $\Theta(logn)$ tight bound.  Sorry for the long comment, does that sound right?

Reply:
Ok, so it's about the different types of asymptotic complexities? That sounds like it's going in the right direction.